recipes/hdf5/all/conanfile.py

```diff
@@ -63,12 +66,15 @@ def validate(self):
                 self.info.options.szip_encoding and \
                 not self.dependencies["szip"].options.enable_encoding:
             raise ConanInvalidConfiguration("encoding must be enabled in szip dependency (szip:enable_encoding=True)")
+        if self.settings.get_safe("compiler.cppstd"):
+            check_min_cppstd(self, self._v1_14_minimum_cpp_standard)
```

Comment:
But C++11 is not required before 1.14?
I would rename this `_v1_14_minimum_cpp_standard` to `_min_cppstd`, and return 98 is < 1.14, 11 otherwise.